song/app.js

```diff
@@ -31,6 +31,22 @@ app.get("/ping", function (req, res, next) {
   res.json({ message: "pong" });
 });
 
+app.post("/join", async (req, res, next) => {
+  const { id, name, email, profile_image, password } = req.body;
```

Comment:
```suggestion
  const { name, email, profileImage, password } = req.body;
```
1. id는 database에서 고유한 pk값으로 부여하고 관리하는 값입니다. user가 회원가입을 할 때, user는 당연하고 client(프론트엔드)에서도 어떠한 값이 유효한지 전혀 알 수 없습니다. id값은 받지 않도록 해주세요~
2. javascript에서는 conding convention으로 snake_case(밑줄로 이어주는)를 쓰지 않고 있습니다. camelCase(소문자로 시작하여 대문자를 섞는)를 활용하여 수정해주세요!